File: migrations/1749567574_updated_store_options.go
package migrations

import (
	"encoding/json"

	"github.com/pocketbase/pocketbase/core"
	m "github.com/pocketbase/pocketbase/migrations"
)

func init() {
	m.Register(func(app core.App) error {
		collection, err := app.FindCollectionByNameOrId("pbc_537519633")
		if err != nil {
			return err
		}

		// update collection data
		if err := json.Unmarshal([]byte(`{
			"createRule": "",
			"updateRule": ""
		}`), &collection); err != nil {
			return err
		}

		return app.Save(collection)
	}, func(app core.App) error {
		collection, err := app.FindCollectionByNameOrId("pbc_537519633")
		if err != nil {
			return err
		}

		// update collection data
		if err := json.Unmarshal([]byte(`{
			"createRule": "@request.auth.id != '' && @request.auth.collectionName = 'users' &&  @request.auth.isManager = true",
			"updateRule": "@request.auth.id != '' && @request.auth.collectionName = 'users' &&  @request.auth.isManager = true"
		}`), &collection); err != nil {
			return err
		}

		return app.Save(collection)
	})
}

File: pb_migrations/1749567574_updated_store_options.js
/// <reference path="../pb_data/types.d.ts" />
migrate((app) => {
  const collection = app.findCollectionByNameOrId("pbc_537519633")

  // update collection data
  unmarshal({
    "createRule": "",
    "updateRule": ""
  }, collection)

  return app.save(collection)
}, (app) => {
  const collection = app.findCollectionByNameOrId("pbc_537519633")

  // update collection data
  unmarshal({
    "createRule": "@request.auth.id != '' && @request.auth.collectionName = 'users' &&  @request.auth.isManager = true",
    "updateRule": "@request.auth.id != '' && @request.auth.collectionName = 'users' &&  @request.auth.isManager = true"
  }, collection)

  return app.save(collection)
})
